sqldelight-compiler/dialect/src/main/kotlin/app/cash/sqldelight/dialect/api/SqlDelightDialect.kt

```diff
@@ -42,4 +42,6 @@ interface SqlDelightDialect {
    * Called when this dialect should initialize its parser.
    */
   fun setup()
+
+  fun connectionManager(): ConnectionManager? = null
```

Comment:
Should this be a `val`? Each invocation creates a new instance but it seems like a stateful instance where you would want to return the same one each time.